uPortal-webapp/src/main/resources/properties/security.properties

```diff
@@ -38,14 +38,14 @@ portal.context=/uPortal
 portal.protocol.server.context=${portal.protocol}://_CURRENT_SERVER_NAME_${portal.context}
 portal.login.url=${portal.protocol.server.context}/Login
 
-# Needed for proxy-cas in load-balanced system like apache to give back the proxy-ticket on the server that requested it
-# it requires that the load-balanced instance has his serverName.
+# Needed for proxy-cas in load-balanced system like Apache to return the
+# proxy-ticket to the requesting server. It requires that the load-balanced
+# instance has this serverName.
```

Comment:
=> ```It requires that the load-balanced instance has his serverName``` warning it's not ***this*** but ***his***

After I can understand my English isn't really good. I meen that in load-balanced system there is a common serverName and each instance has his owned serverName  (ie there is a https://my.univ.edu/ for common serverName and after each tomcat have a different servername like https://portal1.univ.edu/ for the first tomcat, https://portal2.univ.edu/ for the second... etc, it's to be able to come back with the proxy ticket on the server that requested it when we don't have shared cluster).

Reply:
How's this? If it looks good I'll update the commit 

```
# Needed for proxy-cas in load-balanced system like Apache to return the
# proxy-ticket to the requesting server. It requires that the load-balanced
# instance has his serverName
# Example: my.univ.edu
portal.lbServerName=${portal.server}

# All server names values for multi server name management, separator is a space
# This property should be set/overridden in PORTAL_HOME/uPortal.properties
# Example: portal1.univ.edu portal2.univ.edu
portal.allServerNames=${portal.server}
```

Reply:
All is good :+1: